catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -327,6 +350,7 @@
                             !org.apache.commons.cli.*,
                             !org.hibernate.*,
                             !weblogic.jdbc.*,
+                            !org.abego.treelayout.*,
```

Comment:
It looks like `org.abego.treelayout.*` is excluded in the dependencies, does this need to be omitted here too?